### PR TITLE
TIMELINE REFACTOR [pr]

### PR DESCRIPTION
**Pull request by pdb493**

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -23,13 +22,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $hidden = [
-        'password', 'remember_token',
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:26]: REPLY [1, customer]

File: app/User.php [hunk 2]
```diff
@@ -23,13 +22,7 @@ class User extends Authenticatable
...
+        'password', 
+	'remember_token',
     ];
 }
-
-
-
-
```
Comment (pdb493) [inline app/User.php:32]: INLINE [1, customer]

**Review by pdb493 — COMMENTED** [review]
SUMMARY [1, customer]

File: app/User.php (hunk 2 above)
Comment (pbarsallo) [reply app/User.php:32]: REPLY [1, customer 2]

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <title>Laravel</title>
```
Comment (pbarsallo) [inline resources/views/welcome.blade.php:8]: INLINE [1, customer 2]

**Review by pbarsallo — APPROVED** [review]

**Pull request closed without merging**